en/docs/administer/configure-an-external-idp/configure-asgardeo-as-an-external-idp.md

```diff
@@ -15,7 +15,7 @@ https://wso2.com/asgardeo/docs/guides/applications/register-standard-based-app/#
 - Find the Client ID :
```

Comment:
```suggestion
- Find the Client ID:
```

Reply:
We would need the colon for "Find the well-known URL:" as well right?

Reply:
Regarding the term (IDaaS) in the first para, might have to change it to "identity-as-a-service".